test-utils/runtime-tester/src/run_test.rs

```diff
@@ -0,0 +1,127 @@
+use std::fs::File;
+use std::io;
+use std::path::Path;
+use std::sync::Arc;
+use std::time::{Duration, Instant};
+
+use near_chain::{Block, ChainGenesis, Provenance, RuntimeAdapter};
+use near_chain_configs::Genesis;
+use near_client::test_utils::TestEnv;
+use near_crypto::InMemorySigner;
+use near_primitives::transaction::{Action, SignedTransaction};
+use near_primitives::types::{BlockHeight, Nonce};
+use near_store::test_utils::create_test_store;
+use nearcore::{config::GenesisExt, NightshadeRuntime};
+
+use serde::{Deserialize, Serialize};
+
+pub fn run_from_json(path: &str) -> Result<RuntimeStats, io::Error> {
```

Comment:
Slightly more than a nit: let's keep the types strong! 

```suggestion
pub fn run_from_json(path: &Path) -> Result<RuntimeStats, io::Error> {
```